### ISSUE TITLE
YouTube video taken down

### ISSUE THREAD

**Issue by x0lani** [LBBO/netflix-migrate#59]
The YouTube video link in README.md is no longer available. 

The message reads "This video has been removed for violating YouTube's policy on spam, deceptive practices, and scams."
They're on to you... :)

**Comment by LBBO** [comment]
Huh, interesting. Can you explain this, @anjumrafidofficial?

**Comment by anjumrafidofficial** [comment]
@LBBO Yeah seems like YouTube don't like our video. I've appealed 5 days ago & they reject it.

![Screenshot_20210505-190255_Gmail](https://user-images.githubusercontent.com/64251776/117145329-dbf6f900-add4-11eb-964f-044a327fb87e.png)

**Comment by natts1** [comment]
Howabout putting it on Vimeo or elsewhere?

**Comment by ChunkyCookie88** [comment]
at least like reupload it. its obviously because of the title. the title looks extremely spammy and they were right to have taken it down
